### DISCUSSION TITLE
Add and access UI elements?

### DISCUSSION THREAD

**Discussion by rohit-kumar-j** [google-deepmind/mujoco#369] [Asking for Help] [answered]
How do we add a slider to the UI in python similar to the one in simulate.cc? I have read the [basic-usage](https://mujoco.readthedocs.io/en/latest/python.html#basic-usage) section of the Python bindings but cannot figure out the UI init and access.

```python
# init ui?
self.ui = mujoco.MjUi() # not working 
```

**Comment by yuvaltassa** [comment, marked as answer]
Hi,

The `mjUI` API is currently missing from the Python bindings.

Please open an "enhancement" issue and we will address this soon.

Thanks

**Reply by rohit-kumar-j** [reply]
Thanks! [opened](https://github.com/deepmind/mujoco/issues/370)

**Comment by kshfire** [comment]
I'm looking forward to the mjUI() API, too.
Because I am in confusion when I try to use the UI framework functions to the Python codes. (I couldn't find the mjUI API in the python)
